### ISSUE TITLE
Cannot select date prior to July 1st 1937 (Dutch locale)

### ISSUE THREAD

**Issue by srijken** [vaadin/vaadin-date-picker#596]
### Description
When selecting a date prior to July 1st 1937 (for example any day in June 1937) nothing happens

This could be related to the 28 second increase described here:
https://www.timeanddate.com/time/change/netherlands/amsterdam?year=1937

The exact timezone was GMT +0h 19m 32.13s until July 1, 1937, when it was simplified to GMT +0h 20m.
https://en.wikipedia.org/wiki/UTC%2B00:20

This results in the behavior below. Note the very weird timezone offset
```
new Date('1937-06-22')
11:29:20.836 Tue Jun 22 1937 01:19:32 GMT+0119 (Midden-Europese zomertijd)
```

### Expected outcome
Date selection works as normal

### Actual outcome
Nothing gets selected

### Steps to reproduce
1. Put a `vaadin-date-picker` element in the page.
2. Open the page in a web browser.
3. Click the input of `vaadin-date-picker` element.
4. Scroll to 1937
5. Click any date before 1st of July

### Browsers Affected
- [x] Chrome
- [ ] Firefox
- [x] Safari
- [ ] Edge
- [ ] IE 11
- [x] iOS Safari
- [x] Android Chrome

**Comment by tomivirkki** [comment]
Hi @srijken, which version of `<vaadin-date-picker>` do you use? And did you install from Bower or npm?

**Comment by srijken** [comment]
This is both on https://vaadin.com/components/vaadin-date-picker/html-examples/date-picker-basic-demos and in our app that uses v2.0.7 from bower.

**Comment by manolo** [comment]
@srijken do you mind to check if the problem still happens in master using this url https://cdn.vaadin.com/vaadin-date-picker/3.2.0-alpha2/demo/#date-picker-basic-demos ? we fixed some issues in 3.1.2

**Comment by srijken** [comment]
@manolo  Works fine in that version, we're currently upgrading to v3.1.2

**Comment by manolo** [comment]
Thank you for the feed back, closing this.

**Comment by web-padawan** [comment]
@srijken FYI, this should be also fixed in `3.0.2` but apparently https://vaadin.com still uses `3.0.1`

**Comment by manolo** [comment]
To be more specific this issue was fixed in https://github.com/vaadin/vaadin-date-picker/commit/914e834b370a7300e7c34e6778f21203c9bd1d86 commit in master, thus first version without the issue was `3.1.1`.
But it also was cherry-picked in `3.0.x` branch in https://github.com/vaadin/vaadin-date-picker/commit/d9fb68c34f948ca4f670e1217a97571e9addd51c hence those using this branch need to update to `3.0.2`
